app/views/feeds/_form_main.html.erb

```diff
@@ -0,0 +1,203 @@
+<div class="col-12 mb-4 prx-field-group">
+  <div class="form-floating">
+    <%= form.text_field :feed_link, value: feed.published_url, disabled: true %>
+    <%= form.label :feed_link %>
+  </div>
+
+  <%= link_to(feed.published_url, class: "prx-btn-help", target: :_blank) do %>
```

Comment:
For private feeds, this should include a valid FeedToken.  (See `FeedToken.feed_published_url_with_token`)